### PR TITLE
Adding info for dap codes

### PR DESCRIPTION
**Pull request by geramirez**

### CODE REVIEW

**Comment by geramirez** [comment]
Sorry about all the changes, my editor removes trailing spaces and the end of lines.

File: README.md [hunk 1]
```diff
@@ -36,35 +36,47 @@ ga('set', 'anonymizeIp', true);
...
+The central URLs for the Digital Analytics Program JS snippet are:
+
+* `https://dap.digitalgov.gov/Universal-Federated-Analytics.js` (full)
+* `https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js` (minified)
+
+Agencies are encouraged to use the following HTML snippet to participate in the Digital Analytics Program:
+
+```html
+<!-- We participate in the US government's analytics program. See the data at analytics.usa.gov. -->
+<script id="_fed_an_ua_tag" src="https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js"></script>
```
Comment (konklone) [inline README.md]: I have a PR open for the DAP -- https://github.com/digital-analytics-program/gov-wide-code/pull/27 -- which adds the `async` attribute to the `<script>` tag. GA uses `async` for their own snippet already.

Comment (geramirez) [inline README.md]: Fixed.

**Comment by gbinal** [comment]
Looks good - merging

**Pull request merged**